src/cmd/cmd.cpp

```diff
@@ -331,6 +331,7 @@ CmdOptions parseOptions(const QStringList &app_args)
     auto maxRetriesOption = addOption({ { QStringLiteral("max-sync-retries") }, QStringLiteral("Retries maximum n times (default to 3)"), QStringLiteral("n") });
     auto uploadLimitOption = addOption({ { QStringLiteral("uplimit") }, QStringLiteral("Limit the upload speed of files to n KB/s"), QStringLiteral("n") });
     auto downloadLimitption = addOption({ { QStringLiteral("downlimit") }, QStringLiteral("Limit the download speed of files to n KB/s"), QStringLiteral("n") });
+    auto syncHiddenFilesOption = addOption({ { QStringLiteral("H"), QStringLiteral("sync-hidden-files") }, QStringLiteral("Enables synchronization of hidden files") });
```

Comment:
Please remove the support for -H... case sensitive arguments 😵...

Reply:
It's pretty much standard in unix tools, `-H` is used very frequently to, e.g., specify an alternative home/config directory. Case insensitivity is only a thing in Windows, really.